src/Symfony/Component/JsonPath/Test/JsonPathAssertionsTrait.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonPath\Test;
+
+use Symfony\Component\JsonPath\JsonCrawler;
+use Symfony\Component\JsonPath\JsonPath;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ *
+ * @psalm-require-extends \PHPUnit\Framework\Assert
+ *
+ * @experimental
+ */
+trait JsonPathAssertionsTrait
+{
+    public static function assertJsonPathEquals(mixed $expectedValue, JsonPath|string $jsonPath, string $json, string $message = ''): void
```

Comment:
Every assertions methods in PHPUnit are `final` and has the `@throws ExpectationFailedException` annotation. I think we can do the same so that we can change the signature later if necessary.

Reply:
Updated as well, thanks 👍